lib/plugin.ts

```diff
@@ -0,0 +1,31 @@
+import tls = require('tls');
+import apiSession = require('./api-session');
+import conf = require('./config');
+
+export = Plugin;
+
+class Plugin {
+
+    // Custom logging plugin.
+    // Log client cert details, request body, etc
+    public static log(): (req: apiSession.OurRequest,
+                          res: apiSession.OurResponse,
+                          next: Function ) => void {
+
+        return function log( req: apiSession.OurRequest,
```

Comment:
Let's remove the space between the opening parens and the parameter. => `log(req:`